src/assets/css/app.css

```diff
@@ -78,7 +78,7 @@
     'Helvetica Neue', 'Noto Sans', 'Liberation Sans', Arial, sans-serif,
     'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
   --bs-font-monospace: SFMono-Regular, Menlo, Monaco, Consolas,
-    'Liberation Mono', 'Courier New', monospace;
+    --bs-body-color 'Liberation Mono', 'Courier New', monospace;
```

Comment:
1. Exclude this file from the PR. It's not a requirement of the issue
1. Use CSS exclusively from `src/style/app.module.css`,